app/controllers/taxi_requests_controller.rb

```diff
@@ -11,4 +11,27 @@ def index
 
     json_success taxi_requests.map { |request| TaxiRequestSerializer.new(request).serialized_json }
   end
+
+  def create
+    params = create_param
+    pending_request = TaxiRequest.find_by(status: TaxiRequest.statuses[:pending])
+    raise Exceptions::Forbidden, '승객만 배차 요청할 수 있습니다' unless current_user.passenger?
+    raise Exceptions::Conflict, '아직 대기중인 배차 요청이 있습니다' unless pending_request.nil?
```

Comment:
코드스타일 차이일 수도 있을 것 같습니다만 falsy값인지 체킹한 후 그 결과가 다시 falsy한지 체킹하는 것보다는 if로 표현하는게 더 직관적이지 않을까요?

```suggestion
    raise Exceptions::Conflict, '아직 대기중인 배차 요청이 있습니다' if pending_request.present?
```

Reply:
넵 해당 부분도 수정하여 반영하였습니다 :)